components/Common/ThemeToggle/index.tsx

```diff
@@ -0,0 +1,38 @@
+import { MoonIcon, SunIcon } from '@heroicons/react/24/outline';
+import { AccessibleIcon } from '@radix-ui/react-accessible-icon';
+import { useTheme } from 'next-themes';
+import type { FC } from 'react';
+import { useIntl } from 'react-intl';
+
+import styles from './index.module.css';
+
+const ThemeToggle: FC = () => {
+  const intl = useIntl();
+  const { theme, setTheme } = useTheme();
+
+  const isDark = theme === 'dark';
+
+  const toggleTheme = () => setTheme(isDark ? 'light' : 'dark');
+
+  return (
+    <button
+      type="button"
+      onClick={toggleTheme}
+      onKeyDown={toggleTheme}
+      className={styles.themeToggle}
+    >
+      <AccessibleIcon
+        label={intl.formatMessage({
+          id: 'components.header.buttons.toggleDarkMode',
+        })}
+      >
+        <>
```

Comment:
this is done because without the fragment it's create error boundary.
It's because radix use `React.cloneElement`
[source code](https://github.com/radix-ui/primitives/blob/c31c97274ff357aea99afe6c01c1c8c58b6356e0/packages/react/accessible-icon/src/AccessibleIcon.tsx#L15-L27)

Reply:
No, you're wrong. The issue is that AccessibleIcon should have only a single children.

So you should wrap AccessibleIcon on each one of these Icon Elements. By adding a `<>` you are pretty much destroying all the accessibility benefits this component creates.

Reply:
@bmuenzenmeyer based on your comment on the LanguageDropdown PR. Should AccessibleIcon wrap the Button element or each Icon element?

Also on that, should AccessibleIcon handle the onClick or onKeyPresses or what?

Reply:
JAWS currently announces https://64c7d71358830e9105808652-bhljmtpybt.chromatic.com/?path=/story/common-themetoggle--default correctly